src/main/java/com/github/complate/ComplateViewResolver.java

```diff
@@ -20,15 +20,19 @@
     public ComplateViewResolver(final ScriptingEngine scriptingEngine,
                                 final Resource bundle) {
         Assert.notNull(scriptingEngine, "ScriptingEngine must not be null");
-        Assert.notNull(bundle,
-            "bundle must not be null");
+        Assert.notNull(bundle, "bundle must not be null");
         this.scriptingEngine = scriptingEngine;
         this.bundle = bundle;
     }
 
+    public ComplateViewResolver(final Resource bundle) {
+        Assert.notNull(bundle, "bundle must not be null");
```

Comment:
I personally dislike more than one constructor which does real work if possible. Therefore I would implement this one as:
`this(new NashornScriptingBridge(), bundle);`

Because we are at such a low version we could also think about reverting the order of the two parameters because that would fit the normal java overloading conventions.